explainer.md

```diff
@@ -572,6 +572,24 @@ const cookie = await cookieStore.get('session-id');
 console.log(cookie);
 ```
 
+#### Read / Write Race
+
+Cookie store operations are not an atomic read-modify-write transaction. Failure to await during an operation can cause unexpected results as the following.
```

Comment:
The ability of passing the return value of `cookieStore.set()` to `cookieStore.get()` may lead to code that reads like an atomic read-modify-write operation, when it reality it's not.